tests/helpers.py

```diff
@@ -61,6 +61,18 @@ def __ne__(self, other: object) -> bool:
 assert Missing() != Missing()
 
 
+def get_captcha_from_session_register(client: FlaskClient) -> str:
```

Comment:
Could just speed up tests and set `math_answer` to a hard coded variable for testing and then just always set that in requests to that endpoint. Or, could just disable CAPTCHAs for testing.